emails/models.py

```diff
@@ -520,6 +520,12 @@ def is_flagged(self):
         # user was flagged and the premium feature pause period is not yet over
         return True
 
+    @property
+    def metrics_enabled(self) -> bool:
+        if self.fxa:
+            return bool(self.fxa.extra_data.get("metricsEnabled", True))
+        return True
```

Comment:
praise: sensible defaults.

Reply:
This could probably use a comment, so future engineers can see why we think it should default to enabled.